### PR TITLE
NotificationPanel fix for not showing rendered messages.

### LINKED ISSUE #723: NotificationPanel does not respect value set by showRenderedMessages(boolean showRenderedMessages)
If you call `showRenderedMessages(final boolean showRenderedMessages)` on notification panel, value is set, but too late. 

This code
```
@Override
    protected FeedbackMessagesModel newFeedbackMessagesModel() {
        FeedbackMessagesModel model = super.newFeedbackMessagesModel();

        if (!showRenderedMessages) {
            model.setFilter(new IFeedbackMessageFilter() {
                @Override
                public boolean accept(FeedbackMessage message) {
                    return !message.isRendered();
                }
            });
        }

        return model;
    }
```
is called from the constructor meaning value of `showRenderedMessages` is always false.

### PR DESCRIPTION
**Pull request by vrozkovec**
newFeedbackMessagesModel() is called from the constructor
and because of that calling
showRenderedMessages(boolean) later had no effect

Fixes #723

### CODE REVIEW

File: bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/common/NotificationPanel.java [hunk 1]
```diff
@@ -18,8 +18,16 @@
 public class NotificationPanel extends FencedFeedbackPanel {
 
     private Duration duration;
-    private boolean showRenderedMessages = false;
-
+	private IFeedbackMessageFilter SKIP_RENDERED_MESSAGES_FILTER = new IFeedbackMessageFilter()
```
Comment (martin-g) [inline bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/common/NotificationPanel.java]: Please add `static final`

Comment (vrozkovec) [reply bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/common/NotificationPanel.java]: added

**Comment by martin-g** [comment]
Thank you, @vrozkovec !

**Comment by martin-g** [comment]
Cherry picked it to 7.x and 8.x branches!

**Pull request merged**